src/export/export.js

```diff
@@ -176,8 +176,8 @@ module.exports = (router) => {
                   );
                 }
                 else {
-                  if (req.encryptedComponents && Object.keys(req.encryptedComponents).includes(key) && field.buffer) {
```

Comment:
I'm suspicious of this change _only_ because [other decrypt routines](https://github.com/formio/formio-server/blob/7dd5b55b3deacdf273861f40717aea63c069d76f/src/util/util.js#L87) that we use contain a check for buffer-like properties before proceeding. Can you explain why this situation differs?

Reply:
the way I see it is we have 2 ways to pass data to decrypt function, just the regular data object, or the buffer field inside of it (i.e. formio-server encryptDecrypt passes a data object to decrypt function). In this case, I made a change so that if the server can't decrypt data (in case we don't have a secret or the project isn't loaded), the returned data would be in form of hashed value instead of the buffer array